engineering_tools/carma

```diff
@@ -116,22 +151,44 @@ carma-config__reset() {
 }
 
 carma-config__list_local() {
+    if [ "$1" = "-d" ]; then
+        local USERNAME=usdotfhwastoldev
+    else
+        local USERNAME=usdotfhwastol
+    fi
     echo "Locally installed images: "
     echo ""
-    docker images usdotfhwastol/carma-config
+    docker images $USERNAME/carma-config
 }
 
 carma-config__list_remote() {
-    echo "Remotely available images from usdotfhwastol Dockerhub: "
-    echo ""
-    echo "IMAGE                           TAG"
-    wget -q https://registry.hub.docker.com/v1/repositories/usdotfhwastol/carma-config/tags -O -  | \
-    sed -e 's/[][]//g' -e 's/"//g' -e 's/ //g' | tr '}' '\n'  | \
-    awk -F: '{print "usdotfhwastol/carma-config\t"$3}'
+    if [ "$1" = "-d" ]; then
```

Comment:
This one I'm less sure of, I do like the flag here so people don't get confused as to what configs should be installed (the released ones vs the dev/candidate images).